src/common/feature-flags.ts

```diff
@@ -75,6 +76,15 @@ export function getAllFeatureFlagDetails(): FeatureFlagDetail[] {
             isPreviewFeature: false,
             forceDefault: true,
         },
+        {
+            id: FeatureFlags.newIssueFilingExperience,
+            defaultValue: false,
+            displayableName: 'New issue filing experience',
+            displayableDescription:
+                'Enable the new issue filing experience. Needs "Issue filing" preview feature to be enable in order to work',
```

Comment:
>Issue filing [](start = 64, length = 12)

shouldnt we remove this feature flag?

Reply:
nop, 2 different purposes and out of scope for our current feature